src/repl/query_print.rs

```diff
@@ -235,18 +235,19 @@ fn write_query_result_<R: QueryPrinter>(
 
             match &meta {
                 MetaValue {
-                    priority: MergePriority::Default,
+                    priority: MergePriority::Bottom,
                     value: Some(t),
                     ..
                 } if selected_attrs.default => {
                     renderer.write_metadata(out, "default", &t.as_ref().shallow_repr())?;
                     found = true;
                 }
                 MetaValue {
-                    priority: MergePriority::Normal,
+                    priority: MergePriority::Numeral(n),
                     value: Some(t),
                     ..
                 } if selected_attrs.value => {
+                    renderer.write_metadata(out, "priority", &format!("{}", n))?;
```

Comment:
an improvement could be to check 0 equality and don't print anything if it's the case

Reply:
Not really, because `0.0` may have been written down. We could check for `Neutral` indeed. On the other hand, it can be useful to know the actual priority, even if it hasn't been annotated?